### PR TITLE
Big Random cleanup

### PR DESCRIPTION
**Pull request by kamilritz**
And another one. Should be reviewed commit by commit.

Firmware tests getting fixed in https://github.com/PX4/ecl/pull/865

### CODE REVIEW

File: EKF/covariance.cpp [hunk 1]
```diff
@@ -848,6 +848,20 @@ void Ekf::fixCovarianceErrors(bool force_symmetry)
 	}
 }
 
+// if the covariance correction will result in a negative variance, then
+// the covariance matrix is unhealthy and must be corrected
+bool Ekf::isCovarianceUpdateHealthy(const SquareMatrix24f& KHP) {
```
[the remark this answers is not in the document]
Comment (kamilritz) [reply EKF/covariance.cpp:853]: Good point. Will use `checkAndFixCovarianceUpdate`

File: EKF/ekf.h [hunk 2]
```diff
@@ -380,7 +380,7 @@ class Ekf : public EstimatorInterface
 	bool _mag_decl_cov_reset{false};	///< true after the fuseDeclination() function has been used to modify the earth field covariances after a magnetic field reset event.
 	bool _synthetic_mag_z_active{false};	///< true if we are generating synthetic magnetometer Z measurements
 
-	bool _yaw_use_inhibit{false};		///< true when yaw sensor use is being inhibited
+	bool _yaw_fusion_is_inhibited{false};		///< true when yaw sensor use is being inhibited
```
[the remark this answers is not in the document]
Comment (kamilritz) [reply EKF/ekf.h:383]: I also thought about this. Will switch to `is_yaw_fusion_inhibited`

**Comment by bresch** [comment]
@kamilritz there are 4 commits that add "missing consts". Could you squash them? Or do you want to squash the whole PR?

**Comment by kamilritz** [comment]
@bresch let me see, if I can squash them. I would advise against squashing the whole PR, single commits are reasonable and it helps to bisect.

**Comment by kamilritz** [comment]
@bresch I managed to squash 3 of them together. The last one is not that easy to squash.

**Comment by kamilritz** [comment]
@bresch The branch is updated, but it seems that this PR did not notice the update.

**Pull request merged**